libraries/lib-datahandler/src/main/thrift/codescoop.thrift

```diff
@@ -98,7 +103,7 @@ struct CodescoopComponent {
     3: required string owner,
     4: required string name,
     5: required string purl,
-    6: required i32 type,
+    6: required string type,
```

Comment:
I think thrift requests, that one should also change the number in the beginning to a number that is new, e.g.
```suggestion
    // 6: required i32 type, // type changned to string
    9: required string type,
```